docs/acquisition.rst

```diff
@@ -0,0 +1,160 @@
+that can be acquired while maintaining reasonable temporal resolution.
+As such, ``tedana`` allows users to provide a T2* map as input to the workflow,
+which means that it may be beneficial to acquire a quantitative T2* map if you
+are also acquiring ME-fMRI data.
+
+Quantitative T2* mapping can be done with a multi-echo GRE sequence, such as a
+multi-echo FLASH stock sequence, with a large number of echoes (e.g., 12).
+When acquiring such a scan, it is best to reconstruct both magnitude and phase data.
+
+
+Acquisition parameter recommendations
+-------------------------------------
+There is no empirically tested best parameter set for multi-echo fMRI acquisition.
+The guidelines for optimizing parameters are similar to single-echo fMRI.
+For multi-echo fMRI, the same factors that may guide priorities for single echo
+fMRI sequences are also relevant.
+Choose sequence parameters that meet the priorities of a study with regards to spatial resolution,
+spatial coverage, sample rate, signal-to-noise ratio, signal drop-out, distortion, and artifacts.
+
+A minimum of 3 echoes is required for running the current implementation fo TE-dependent denoising in
+``tedana``.
+It may be useful to have at least one echo that is earlier and one echo that is later than the
+TE one would use for single-echo T2* weighted fMRI.
+
+.. note::
+    This is in contrast to the **dual echo** denoising method which uses a very early (~5ms)
+    first echo in order to clean data. For more information on this method, see `Bright and Murphy`_ (2013).
+
+.. _Bright and Murphy: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3518782/
+
+More than 3 echoes may be useful, because that would allow for more accurate
+estimates of BOLD and non-BOLD weighted fluctuations, but more echoes have an
+additional time cost, which would result in either less spatiotemporal coverage
+or more acceleration.
+Where the benefits of more echoes balance out the additional costs is an open research question.
+
+We are not recommending specific parameter options at this time.
+There are multiple ways to balance the slight time cost from the added echoes that have
+resulted in research publications.
+We suggest new multi-echo fMRI users examine the :ref:`spreadsheet of publications` that use
+multi-echo fMRI to identify studies with similar acquisition priorities,
+and use the parameters from those studies as a starting point.
+More complete recommendations
+and guidelines are discussed in the `appendix`_ of Dipasquale et al, 2017.
+
+.. _appendix: https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0173289
+
+.. note::
+    In order to increase the number of contrasts ("echoes") you may need to first increase the TR, shorten the
+    first TE and/or enable in-plane acceleration.
+    For typically used parameters see the `parameters and publications page`_
+.. _parameters and publications page: https://tedana.readthedocs.io/en/latest/publications.html
```

Comment:
publications.html is now a dead link. Once there's an agreement where this should be, update the link.